api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/security/SecurityConfiguration.java

```diff
@@ -97,8 +102,8 @@ public SecurityFilterChain basicAuthOrTokenOrCertApiDocFilterChain(HttpSecurity
             mainframeCredentialsConfiguration(
                     baseConfiguration(http.requestMatchers(matchers -> matchers.antMatchers(APIDOC_ROUTES, STATIC_REFRESH_ROUTE)))
```

Comment:
I guess this can be replaced as well...
```suggestion
                    baseConfiguration(http.securityMatchers(matchers -> matchers.requestMatchers(APIDOC_ROUTES, STATIC_REFRESH_ROUTE)))
```